shared/actions/fs/platform-specific.desktop.js

```diff
@@ -315,38 +314,37 @@ const openAndUpload = (state, action) =>
   )
 
 const loadUserFileEdits = (state, action) =>
-  RPCTypes.SimpleFSSimpleFSUserEditHistoryRpcPromise().then(writerEdits => {
-    const tlfUpdates = Constants.userTlfHistoryRPCToState(writerEdits || [])
-    const updateSet = tlfUpdates
-      .reduce(
-        (acc: I.Set<Types.Path>, u) =>
-          Types.getPathElements(u.path).reduce((acc, e, i, a) => {
-            if (i < 2) return acc
-            const path = Types.getPathFromElements(a.slice(0, i + 1))
-            return acc.add(path)
-          }, acc),
-        I.Set()
-      )
-      .toArray()
-    // TODO (songgao): make a new action that accepts an array of updates,
-    // so that we only need to trigger one update through store/rpc/widget
-    // for all these each time.
-    return [
-      ...updateSet.map(path =>
-        FsGen.createFilePreviewLoad({
-          identifyBehavior: RPCTypes.tlfKeysTLFIdentifyBehavior.chatGui,
-          path,
-        })
-      ),
-      FsGen.createUserFileEditsLoaded({tlfUpdates}),
-    ]
-  })
+  RPCTypes.SimpleFSSimpleFSUserEditHistoryRpcPromise().then(writerEdits =>
+    FsGen.createUserFileEditsLoaded({
+      tlfUpdates: Constants.userTlfHistoryRPCToState(writerEdits || []),
+    })
+  )
```

Comment:
I thought we needed to do `SimpleFSStat` for each in edit history in order to properly display icons in the widget, but looks like not. My guess is we needed to have the thing in `state.fs.pathItems` before try to display. That has changed since we had KBFS links in chat, and now we just load `PathItem` on-demand. So just get rid of this.